sql_metadb/derived_tables/runlist.txt

```diff
@@ -39,3 +39,4 @@ item_ext.sql
 loans_items.sql
 loans_renewal_count.sql
 po_lines_tags.sql
+users_groups.sql
```

Comment:
Since this query depends on the derived table users_departments_unpacked you should make sure the entry in the runlist is before this query. I know it was not there when this PR was opened but to be sure the runlist.txt is merged correctly you could pull the current `main` branch  into to your `users_groups` branch. Thanks!

Reply:
Thank you for your feedback! Unfortunately, when I try to pull the current main branch into my users_groups branch, GitHub just tells me to open another PR. Is there a way around that?